third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -223,6 +241,24 @@ std::vector<ComputationClient::DataPtr> PjRtComputationClient::TransferToServer(
   return datas;
 }
 
+ComputationClient::DataPtr PjRtComputationClient::TransferReplicatedDataToServer(
```

Comment:
`TransferShardsToServer` takes in `OpSharding`. Can this be done using `TransferShardsToServer` if `OpSharding` sharding type is replicated?

Reply:
Sounds good, I'll try reusing PjRtShardedData with `OpSharding::Replicated`